features/tab-switcher-animation.json

```diff
@@ -0,0 +1,7 @@
+{
+    "_meta": {
+        "description": "A feature flag that determines whether the tab switcher info panel animation is enabled.",
+        "sampleExcludeRecords": {}
```

Comment:
I don't think this needs this samplee xcluded records? Also missing the default state disabled

Reply:
sorted